pkg/controller/woopra.go

```diff
@@ -18,15 +18,22 @@ func NewWoopra() *Woopra {
 }
 
 // GetHandler returns the woopra-domain for UI.
-func (w *Woopra) GetWoopraDomain(ctx *gin.Context) {
+func (w *Woopra) GetDevSpacesWoopraDomain(ctx *gin.Context) {
```

Comment:
I guess the comment should be updated to `woopra-domain for DevSpaces`?

Reply:
done in [3befcfd](https://github.com/codeready-toolchain/registration-service/pull/323/commits/3befcfd9dcd0c7ea9c2515921c69413c5ef1f927)